cmd/krel/cmd/promote-images.go

```diff
@@ -243,7 +248,7 @@ func runPromote(opts *promoteOptions) error {
 		return errors.Wrap(err, "adding image manifest to staging area")
 	}
 
-	commitMessage := fmt.Sprintf("releng: Image promotion for %s", opts.tag)
+	commitMessage := fmt.Sprintf("releng: Image promotion for %s", opts.tags[0])
```

Comment:
maybe iterate and add all tags?
like `releng: Image promotion for 1.2.3 / 1.3.4`

just a suggestion not blocking at all

Reply:
Good idea :) Done :+1: